app/src/main/res/layout-land/hints_summary.xml

```diff
@@ -10,106 +10,100 @@
       name="isListExpanded"
       type="Boolean" />
 
-     <variable
+    <variable
       name="viewModel"
       type="org.oppia.android.app.hintsandsolution.HintsViewModel" />
   </data>
 
-    <LinearLayout
-      android:id="@+id/hint_summary_container"
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:id="@+id/hint_summary_container"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="32dp"
+    android:orientation="vertical">
```

Comment:
`ConstraintLayout ` don't need orientation.

Reply:
Yeah, didn't notice that `orientation` attribute is there. Removed now.

Reply:
@ShridharGoel Could you also check few other places like FrameLayout in some files, do we need orientation there?

Reply:
@anandwana001 Not in these four files. But there are a few places in the project, where orientation is used but it isn't required. Should I create an issue for that?

Reply:
okay, Thanks for checking. Could you please file a good-first-issue for this, as I don't think a Frame layout needs an orientation?

Reply:
https://github.com/oppia/oppia-android/issues/2237